### ISSUE TITLE
Google Docs not working: Min 1.12.0 on MacOS Mojave 10.14.6 

### ISSUE THREAD

**Issue by pkuksa** [minbrowser/min#884]
When attempting to sign-in, the following error is displayed:

Couldn't sign you in
This browser or app may not be secure. Learn more
Try using a different browser. If you’re already using a supported browser, you can refresh your screen and try again to sign in.

**Comment by PalmerAL** [comment]
This looks like the same issue as #868. Do you have two-factor authentication enabled on your account?

Also, you can try this and see if it works:
1. Go to developer > inspect browser.
2. Press the escape key, click on the icon with the three dots in the panel that appears, and choose "network conditions".
3. Uncheck "select automatically", and choose "firefox mac" from the dropdown.
4. Reload the page and sign in again.

**Comment by pkuksa** [comment]
Thanks for looking into this!

Even though I could not login by changing the user-agent under "network conditions",
I was able to login to Google Docs by explicitly setting a user-agent string in 
/Applications/Min.app/Contents/Resources/app/main.build.js
as Firefox-Mac:
app.userAgentFallback = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:72.0) Gecko/20100101      Firefox/72.0"

However, the Chrome Web store is not working at all, and I cannot install any extensions ('Add to Chrome' button is missing!).
Chrome Web store is not working under any user-agent string I tried so far: 1. default, 2. Chrome-Mac, or 3. Firefox-Mac.
